datacapturing/src/main/java/de/cyface/datacapturing/persistence/MeasurementPersistence.java

```diff
@@ -197,7 +113,13 @@ public void storeData(final @NonNull CapturedData data, final long measurementId
      * @param location The geo location to store.
      * @param measurementIdentifier The identifier of the measurement to store the data to.
      */
-    public void storeLocation(final @NonNull GeoLocation location, final long measurementIdentifier) {
+    public void storeLocation(final @NonNull GeoLocation location, final long measurementIdentifier)
+            throws DataCapturingException {
+        if (location.getTimestamp() == 0L) {
+            throw new DataCapturingException(
+                    "GeoLocations captured without GpsFix. Make sure you only capture data when a fix is available.");
```

Comment:
This came from the 1.0.0 code so I migrated it back here as this should still be the case (timestamp 0L if no GpsFix) and such location data should never be stored, right?